rolemanager/core/models.py

```diff
@@ -148,24 +232,53 @@ def __eq__(self, other) -> bool:
             return False
         return self.message.id == other.message.id and self.channel.id == other.channel.id
 
+    @property
+    def channel(self) -> Optional[discord.TextChannel]:
+        """
+        The channel where the reaction role message is. If the `.message` attribute
+        is not set, `None` will be returned.
+        """
+        if self.message is MISSING:
+            return None
+        return self.message.channel
+
     @classmethod
     def from_data(cls, manager: ReactionRoleManager, *, data: ReactRolePayload) -> ReactionRole:
         """
         Instantiate this class from raw data. This will automatically add persistent
         view to the bot if the trigger type is interaction.
+
+        Raises
+        -------
+        ValueError
+            Channel with provided ID from the data not found.
         """
+        global Button
+        if Button is None:
+            from .views import Button
```

Comment:
This is to temporarily resolve circular import error.
It would be better if we can get rid of this.